README.md

```diff
@@ -32,7 +32,7 @@ For filing bugs, suggesting improvements, or requesting new features, please ope
 
 Please use the following to reach members of the community:
 
-- Gitter: [rook/rook](https://gitter.im/rook/rook) for general project discussions or [rook-dev](https://gitter.im/rook/rook-dev) for development discussions.
+- Slack: Join our [slack channel](https://rook-slackin.herokuapp.com)
```

Comment:
the heokuapp domain is a little odd to me.  is there an easy way around that?  or is that a fairly standard domain for slack sign ups and no one else would think it's odd?

Reply:
We can possibly use our own cname like slack.rook.io but that will require some more work, not sure it's worth it. Maybe later?

Reply:
sure, i don't think it's a big deal, but it did strike me as odd at first glance